cmd/ethkey/README.md

```diff
@@ -0,0 +1,47 @@
+ethkey
+======
+
+ethkey is a simple command-line tool for working with Ethereum keyfiles.
+
+
+# Usage
+
+## generate
```

Comment:
Please use full command as heading, e.g.

```markdown
## `ethkey generate`

Generates a new keyfile.
```

Reply:
Done. Not sure how well that would work with the longer arguments in sign and verify, though. Let me check how GitHub renders it here :)

Reply:
Doesn't look that nice. I'll probably change that to regular text 
```
*`ethkey generate`*
```